microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -158,19 +156,23 @@ After you verified that the animation isn't performing well, the next step is to
 
    The **Animation Frame Fired** event occurs whenever a [requestAnimationFrame() callback](https://developer.mozilla.org/docs/Web/API/window/requestAnimationFrame) is run.
 
-1. Click the **Animation Frame Fired** event.  The **Summary** panel now shows you information about that event.  Note the **Reveal** link.  After you click it, DevTools highlights the event that initiated the **Animation Frame Fired** event.  Also, focus on the **app.js:96** link.  After you click it, the relevant line in the source code is displayed.
+1. Click the **Animation Frame Fired** event.  The **Summary** panel now shows you information about that event.  Note the **Reveal** link.  After you click it, DevTools highlights the event that initiated the **Animation Frame Fired** event.  Also, focus on the **app.js:125** link.  After you click it, the relevant line in the source code is displayed.
```

Comment:
```suggestion
```suggestion
1. Click the **Animation Frame Fired** event.  The **Summary** panel displays information about that event.

   ![More information about the Animation Frame Fired event](./index-images/animation-frame-fired.png)

1. Click the **Reveal** link.  DevTools highlights the event that initiated the **Animation Frame Fired** event.

1. Click the **app.js:125** link.  The relevant line of source code is displayed.
```
If we're telling user to definitely do this action, seems a separate numbered step.  or two.

This change also moves away from a multi-sentence prose paragraph to numbered action steps, more linear and digestible.

Rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/?branch=pr-en-us-2814#find-the-bottleneck - step 5